### PR TITLE
Archive

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (122)
Screenshot (123)

Features added in this PR :

- Move note from Home Page and show it on Archive Page.
- Delete note from Archive Page.
- Restore note from Archive Page.

### CODE REVIEW

File: src/note-components/Archive/archive.jsx [hunk 1]
```diff
@@ -3,17 +3,73 @@ import "../Homepage/homepage.css"
...
-  const { archive } = noteState;
+const { noteState, noteDispatch } = useNote();
+const { note, trash, archive } = noteState;
+const {authState} = useAuth();
+const { token } = authState;
+
+const addToTrashFromArchive = async (item) => {
+  try{
+  const response = await axios({
+  method: "DELETE",
```
Comment (amartya-rs) [inline src/note-components/Archive/archive.jsx:19]: Please give proper indentation. You can use a formatter like Prettier.

File: src/public-css/root.css [hunk 2]
```diff
@@ -5,4 +5,11 @@
 
 body{
     font-family: 'Montserrat', sans-serif;
+}
+
+.page-heading{
+    color: var(--dark-red);
+    /* margin: -1.313rem; */
+    /* font-size: 4.375rem; */
```
Comment (amartya-rs) [inline src/public-css/root.css:13]: remove comments if not required.

**Review by amartya-rs — APPROVED** [review]
Good to merge with the suggestions.

**Review by Jia-A — APPROVED** [review]
Good naming conventions and nice use of variables throughout the code.
Everything looks fine, you can merge it.

File: src/note-components/Archive/archive.jsx (hunk 1 above)
Comment (Megha-Joshi) [reply src/note-components/Archive/archive.jsx:19]: Okay will do that.

File: src/public-css/root.css (hunk 2 above)
Comment (Megha-Joshi) [reply src/public-css/root.css:13]: Okay

**Pull request merged**